dali/python/nvidia/dali/plugin/jax.py

```diff
@@ -110,6 +110,9 @@ class DALIGenericIterator(_DaliBaseIterator):
     prepare_first_batch : bool, optional, default = True
                 Whether DALI should buffer the first batch right after the creation of the iterator,
                 so one batch is already prepared when the iterator is prompted for the data
+    sharding : ``jax.sharding.Sharding`` comaptible object that if present will be used to build
+                output jax.Array for each category. If ``None`` iterator returns values compatible
+                with pmaped JAX functions.
```

Comment:
```suggestion
    sharding : ``jax.sharding.Sharding`` comaptible object that, if present, will be used to build a
                output jax.Array for each category. If ``None``, the iterator returns values compatible
                with pmapped JAX functions.
```
Not sure about "pmapped" - double 'p' if it's somehow derived from a verb "to map" (map -> mapped).

Reply:
Done

Reply:
What about the other changes?

Reply:
They weren't there yet :) 
Done